Gemfile.lock

```diff
@@ -1,203 +0,0 @@
-GEM
```

Comment:
@nwolfe @camlow325 The Puppet Labs tradition deliberately goes against the Bundler recommendation because their recommendation is based on the assumption that the "application" is a web app being developed and deployed in a distributed fashion.  Their goal is to keep everyone running the exact same version.

We have a need to run different versions in different situations, particularly when we're trying to track changes in active development in multiple dependencies, e.g. puppet, facter, beaker, etc...

Hence the deliberate breaking away from the recommendation.